frontend/src/components/AcitveMembership/ActiveMembership.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+// @material-ui/core components
+import { Grid } from "@material-ui/core";
+// core components
+import Divider from "@material-ui/core/Divider";
+
+interface Keyable {
+    [key: string]: string
+}
+
+interface Props {
+    planType: string,
+    expiry: string,
+    classes: Keyable
+};
+
+const ActiveMembership: React.FC<Props> = (props) => {
+    var todayDate: any = new Date();
+    let remainingDays = Math.ceil((Date.parse(props.expiry) - todayDate) / (1000 * 3600 * 24))
+
+    return (
+        <div>
+            <Grid container>
+                <Grid item xs={6} sm={6} md={6}>
+                    <h5>{props.planType} Membership</h5>
+                </Grid>
+                {/* the line below should use className={classes.textRight} instead of this */}
+                <Grid item xs={6} sm={6} md={6} className={props.classes.textRight}>
```

Comment:
The classes doesn't seem to make sense. Either `ActiveMembership` takes care of this internall or it should passed some abstract props eg `addtionalClasses` loose or `alignRight: boolean` or `alignment: 'left'|'right'`